build-system/server.js

```diff
@@ -91,6 +96,9 @@ if (noCachingExtensions) {
 if (lazyBuild) {
   middleware.push(lazyBuildExtensions);
   middleware.push(lazyBuildJs);
+  if (argv.extensions || argv.extensions_from) {
```

Comment:
Remove this since you are now always building amp.js

Reply:
Aha, I didn't realize you meant to pre-build `amp.js` independent of `--extensions` of `--extensions_from`. I've reworked things so that it happens.